snmp/datadog_checks/snmp/data/profiles/_base_dell.yaml

```diff
@@ -0,0 +1,264 @@
+# Base profile for DELL devices
+
+extends:
+  - _base.yaml
+  
+metrics:
+  - MIB: INTEL-LAN-ADAPTERS-MIB
```

Comment:
We might want to move metrics from this MIB to their own "mixin", eg `_intel-lan-adapters.yaml`, since they're not related to Dell directly (OTOH it makes sense to have the `IDRAC-MIB` and `DELL-RAC-MIB` metrics here). Then we can also include it in the Poweredge/iDRAC profiles.